app/controllers/votes_controller.rb

```diff
@@ -0,0 +1,22 @@
+class VotesController < ApplicationController
+
+  def create
+    @review = Review.find(params[:review_id])
+    @review.votes.where(:user_id => current_user.id).destroy_all
```

Comment:
Unauthenticated user can't view the button, and if they try to curl in it doesn't work and just gives them an error